docs/HOWTO.md

```diff
@@ -0,0 +1,100 @@
+## Migrations
+
+### How to run migrations
+
+- for `docker` local setup - run `docker compose exec app npm run migrations`
+- for other database locations - run `npm run migrations`
+
+### How to create a migration
+
+Change database schema and run `npm run migrations`.
+
+Consider following when creating migration:
+
+- using meaningful name
+- separate dropping/creating tables and columns
```

Comment:
It would be better to add more information here
- where to place migrations
- what files it consists of
- how it has to be named - e.g. "<timestamp>_<custom_name>"
- how to generate migration - maybe prisma has some script for creating migration.
- how to roll-back migration in case of any errors
 
 I think it has to be required also with migration to create a file for its roll-back.

Reply:
`where to place migrations`
they are generated in default locations, there is no ability to choose target folder

`what files it consists of`
there is single `sql` file, these generally should not be edited manually, what could be documented about it? 🤔 

`how it has to be named - e.g. "_<custom_name>"`
i think prisma automatically applies its formating rules for migration written in plain english

`how to generate migration`
there is one command for generationg and running migrations, i will update doc to explicitly note that point

`how to roll-back migration in case of any errors`
there is no such ability - only reset completely. i will add a point about that too